documentation/user_docs/docs/example_scripts/horace_diagnostics.rst

```diff
@@ -10,8 +10,8 @@ features in your data.
 This will take you through all the steps from generation, spurion
 spotting, and analysis.
 
-Generating data
-===============
+Generating the data
+===================
```

Comment:
This is to avoid a warning that there as a duplicate section title `Generating data` below.